hera_sim/beams.py

```diff
@@ -25,7 +25,7 @@ class PolyBeam(AnalyticBeam):
         Reference frequency for the beam width scaling power law, in Hz.
     """
 
-    def __init__(self, beam_coeffs, spectral_index=0.0, ref_freq=1e8):
+    def __init__(self, beam_coeffs, spectral_index=0.0, ref_freq=1e8, **kwargs):
```

Comment:
If you add `**kwargs`, you'll need to document where exactly those arguments get sent. As a user, I can see that it takes `**kwargs`, but I don't know what arguments I can actually pass or what they do.

Reply:
Here, we used  **kwargs in PolyBeam to use the same dictionary input format as for PerturbedPolyBeam class.

Reply:
But you don't need it. Those `**kwargs`  aren't being used for anything. I can't see a use-case in which you require them to be in the base class? THe kwargs in `PerturbedPolyBeam` get passed through to `PolyBeam`, which makes sense. But the ones in `PolyBeam` don't do anything. You might as well not have them.